code/datums/spells/bloodcrawl.dm

```diff
@@ -149,9 +149,13 @@
 		var/mob/living/simple_animal/demon/slaughter/demon = L
 		demon.devoured++
 		to_chat(victim, "<span class='userdanger'>You feel teeth sink into your flesh, and the--</span>")
+		var/obj/item/organ/internal/regenerative_core/legion/core = victim.get_int_organ(/obj/item/organ/internal/regenerative_core/legion)
+		if(core)
+			core.remove(victim)
 		victim.adjustBruteLoss(1000)
 		victim.forceMove(demon)
 		demon.consumed_mobs.Add(victim)
+		ADD_TRAIT(victim, TRAIT_UNREVIVABLE, "unrevivable")
```

Comment:
```suggestion
		ADD_TRAIT(victim, TRAIT_UNREVIVABLE, "demon")
```
The cause would be something like this. So you can have multiple sources of the same trait active at once

Reply:
👍 Ah got it, wasn't sure what that part was